### PR TITLE
Add extractor for Jetty Client.

### PR DESCRIPTION
**Pull request by rghetia**

### CODE REVIEW

File: contrib/http_jetty_client/src/main/java/io/opencensus/contrib/http/jetty/client/OcJettyHttpClientExtractor.java [hunk 1]
```diff
@@ -0,0 +1,70 @@
...
+  public int getStatusCode(@Nullable Response response) {
+    if (response != null) {
+      return response.getStatus();
+    }
+    return 0;
+  }
+
+  @Override
+  public String getRoute(Request request) {
+    return "";
```
Comment (bogdandrutu) [inline contrib/http_jetty_client/src/main/java/io/opencensus/contrib/http/jetty/client/OcJettyHttpClientExtractor.java]: Add a TODO to make this configurable for the user.

**Comment by songy23** [comment]
Please fix the build.

File: all/build.gradle [hunk 2]
```diff
@@ -12,6 +12,7 @@ def subprojects = [
         project(':opencensus-contrib-exemplar-util'),
         project(':opencensus-contrib-grpc-util'),
         project(':opencensus-contrib-grpc-metrics'),
+        project(':opencensus-contrib-http-jetty-client'),
```
[the remark this answers is not in the document]
Comment (rghetia) [reply all/build.gradle]: ok

File: build.gradle [hunk 3]
```diff
@@ -451,6 +451,7 @@ subprojects {
                  'opencensus-contrib-exemplar-util',
                  'opencensus-contrib-grpc-metrics',
                  'opencensus-contrib-grpc-util',
+                 'opencensus-contrib-http-jetty-client',
```
[the remark this answers is not in the document]
Comment (rghetia) [reply build.gradle]: yes.

File: contrib/http_jetty_client/src/main/java/io/opencensus/contrib/http/jetty/client/OcJettyHttpClientExtractor.java (hunk 1 above)
Comment (rghetia) [reply contrib/http_jetty_client/src/main/java/io/opencensus/contrib/http/jetty/client/OcJettyHttpClientExtractor.java]: added.

**Pull request merged**